src/js/bootstrap.js

```diff
@@ -1,63 +1,114 @@
+import React, { useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import { Provider, useDispatch, useSelector } from 'react-redux';
+import axios from 'axios';
+import { spinUpStore } from './redux-config';
+import RootApp from './App/RootApp';
+import { isBeta } from './utils';
+import { loadModuesSchema } from './redux/actions';
+import Cookies from 'js-cookie';
+import { ACTIVE_REMOTE_REQUEST, CROSS_ACCESS_ACCOUNT_NUMBER } from './consts';
 import auth, { crossAccountBouncer } from './auth';
 import sentry from './sentry';
 import createChromeInstance from './chrome/create-chrome';
 import registerUrlObserver from './url-observer';
-import Cookies from 'js-cookie';
-import { ACTIVE_REMOTE_REQUEST, CROSS_ACCESS_ACCOUNT_NUMBER } from './consts';
 
-const initialAccount = localStorage.getItem(ACTIVE_REMOTE_REQUEST);
-if (Cookies.get(CROSS_ACCESS_ACCOUNT_NUMBER) && initialAccount) {
-  try {
-    const { end_date } = JSON.parse(initialAccount);
-    /**
-     * Remove cross account request if it is expired
-     */
-    if (new Date(end_date).getTime() <= Date.now()) {
-      crossAccountBouncer();
+const initializeAccessRequestCookies = () => {
+  const initialAccount = localStorage.getItem(ACTIVE_REMOTE_REQUEST);
+  if (Cookies.get(CROSS_ACCESS_ACCOUNT_NUMBER) && initialAccount) {
+    try {
+      const { end_date } = JSON.parse(initialAccount);
+      /**
+       * Remove cross account request if it is expired
+       */
+      if (new Date(end_date).getTime() <= Date.now()) {
+        crossAccountBouncer();
+      }
+    } catch {
+      console.log('Unable to parse initial account. Using default account');
+      Cookies.remove(CROSS_ACCESS_ACCOUNT_NUMBER);
     }
-  } catch {
-    console.log('Unable to parse initial account. Using default account');
-    Cookies.remove(CROSS_ACCESS_ACCOUNT_NUMBER);
   }
-}
+};
+
+const libjwtSetup = () => {
+  const libjwt = auth();
+
+  function noop() {}
+
+  libjwt.initPromise.then(() => {
+    libjwt.jwt
+      .getUserInfo()
+      .then((...data) => {
+        sentry(...data);
+      })
+      .catch(noop);
+  });
+
+  return libjwt;
```

Comment:
What if the user is not logged in? Do we want to break the flow and login the user or continue with rest of the calculations? I'd guess breaking the cycle. Let's return directly the result from `libjwt.initPromise`

Reply:
But I use libjwt later on the line 76 in **createChromeInstance**, isn't it a problem?